src/shared/utils.ts

```diff
@@ -95,15 +95,17 @@ export class Utils {
     return metaData;
   }
 
-  public static getEntryPointAndHandlerPath(handler) {
+  public static getEntryPointAndHandlerPath(handler: string) {
     let handlerPath = "handler.js";
     let entryPoint = handler;
     const handlerSplit = handler.split(".");
+    const slashIndex = handler.lastIndexOf("/");
 
     if (handlerSplit.length > 1) {
       entryPoint = handlerSplit[handlerSplit.length - 1];
-      handlerPath = `${handler.substring(0, handler.lastIndexOf("."))}.js`;
+      handlerPath = `${handler.substring(slashIndex + 1, handler.lastIndexOf("."))}.js`;
```

Comment:
Currently, the `function.json` file looks something like this:
```
{
   ...
   scriptFile: {functionName}/index.js
   ...
}
```
This causes the file not to be found because it uses a relative path from `function.json`, which sits alongside `index.js`. This fix simply trims the directory name from the path, resulting in just `index.js`.

@wbreza I know you have an open PR for packaging updates. Not sure if you ran into this at all or if you have any other thoughts